setup/sql/5_3_hotfixes.php

```diff
@@ -188,3 +188,33 @@
 <?php
 	$ilDB->manipulate('UPDATE exc_mem_ass_status SET status='.$ilDB->quote('notgraded', 'text').' WHERE status = '.$ilDB->quote('', 'text'));
 ?>
+<#12>
+	<?php
+	/*
+	* This hotfix removes org unit assignments of user who don't exist anymore
+	* 1) select all user_ids from il_orgu_ua
+	* 2) loop through all user_ids from il_orgu_ua
+	* 3) check if an entry in the usr_data table exists for the user_id
+	*  a) if not delete all entries in il_orgu_ua with this user_id
+	*/
+	$sql = "SELECT user_id FROM il_orgu_ua";
```

Comment:
IMHO this should be possible with exactly one query which would be much more performant. @bseglias could you please have a look at it again? Thanks in advance